WL.VR/mission.sqm

```diff
@@ -359,7 +359,7 @@ class Mission
 					id=18;
 					side="WEST";
 					vehicle="B_Soldier_SL_F";
-					player="PLAY CDG";
+					player="PLAYER COMMANDER";
```

Comment:
It's defined like this so you are able to preview the mission in the editor. There are no problems related to this setting in missions, as they behave the same as playable units. Still if you are not conviced, I can change it in the PR